doc/sphinxman/source/scf.rst

```diff
@@ -119,6 +119,7 @@ which are useful in elucidating the stability and reactivity of the system.
 
 .. index::
    pair: SCF; theory
+.. _`sec:scftheory`:
```

Comment:
```suggestion

.. _`sec:scftheory`:
```
it was probably fine as-was since it built locally for you, but it's safer in Sphinx to be generous with blank lines